cmd/ci-secret-bootstrap/main.go

```diff
@@ -120,6 +122,12 @@ func (o *options) completeOptions(censor *secrets.DynamicCensor, kubeConfigs map
 		return err
 	}
 
+	if o.configUpdaterOnly {
+		logrus.WithField("o.configUpdaterOnly", o.configUpdaterOnly).Info("Removing secrets whose name is not config-updater ...")
```

Comment:
The field on this log line will only ever be true. I think a regular old `logrus.Info` would suffice here.

Reply:
Updated.